.storybook/utils/zeroReports.js

```diff
@@ -57,10 +57,12 @@ export function replaceValuesInAnalytics4ReportWithZeroData( report ) {
 	// eslint-disable-next-line no-unused-vars -- Ignore `rows` and `rowCount` since we're omitting them from the returned report object.
```

Comment:
Please remember to review the notes in **Relevant technical choices** in the PR description.